scripts/us_census/pep/pep_by_srh/process.py

```diff
@@ -1006,6 +1417,18 @@ def process(data_directory):
     Args:
       download_dir: download directory - input files are saved here.
     """
+    input_files = []
+    # Walk through the directory and its subdirectories
+    for root, dirs, files in os.walk(_INPUT_FILE_PATH):
+        for file in sorted(files):  # Sort the files alphabetically
+            file_path = os.path.join(root, file)
+            input_files.append(file_path)
+    # Now `input_files` contains paths to all the files in `_INPUT_FILE_PATH` and its subdirectories
+
+    processed_count = 0
+    total_files_to_process = len(input_files)
+    logging.info(f"No of files to be processed {len(input_files)}")
```

Comment:
in log you can print only the variable "total_files_to_process"

Reply:
Done.